### PR TITLE
Chnage navbar

### PR DESCRIPTION
**Pull request by Hunter2065**
# Checklist:

- [x] Added specs for all new ruby code
- [x] Green rspec run
- [x] Green rubocop
- [x] PR review from teammate

# What?
Created most user-friendly navbar and added gravatar 

## Why?
'cause it's  more esthetic and useful

## How?
Changed some stylesheets and created method 'gravatar_for' for users' default avatars

## Testing?
no

## Screenshots (optional)
image
image

## Anything Else?
no)

### CODE REVIEW

**Review by RMuskovets — APPROVED** [review]
LGTM!

File: app/frontend/stylesheets/header.scss [hunk 1]
```diff
@@ -6,5 +6,29 @@ header {
...
+          ul {
+            text-align: center;
+            background-color: #464E4F;
+          }
+        }
+      }
+    }
+  }
+
+  a { color: var(--bs-light) !important; }
```
Comment (andrjuha01) [inline app/frontend/stylesheets/header.scss:29]: Avoid using important.

File: app/frontend/stylesheets/header.scss [hunk 2]
```diff
@@ -6,5 +6,29 @@ header {
...
+}
+
+.hovc:hover {
+  color: #505050 !important;
```
Comment (andrjuha01) [inline app/frontend/stylesheets/header.scss:33]: Same here

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -1,2 +1,8 @@
 module UsersHelper
+  def gravatar_for(user, options = { size: 40 })
```
Comment (andrjuha01) [inline app/helpers/users_helper.rb:2]: Options is an old fashioned.
Now you can use just
user, size: 40 
As a list of params

File: app/helpers/users_helper.rb [hunk 4]
```diff
@@ -1,2 +1,8 @@
...
+    gravatar_id = Digest::MD5.hexdigest(user.email.downcase)
+    size = options[:size]
```
Comment (andrjuha01) [inline app/helpers/users_helper.rb:4]: Than drop this part, you'll already have size in size var

File: app/views/shared/_header.html.slim [hunk 5]
```diff
@@ -5,8 +5,22 @@ header.d-flex.w-100.flex-column
...
+            ul.dropdown-menu.dropdown-menu-end.smc aria-labelledby="dropdownMenuLink"
+              li
+                = link_to 'Profile', '#', class: 'dropdown-item hovc'
+              li
+                = link_to 'My Places', '#', class: 'dropdown-item hovc'
+              li
+                = link_to 'Sign Out', destroy_user_session_path, method: :delete,
+                                                                 class: 'dropdown-item hovc'
+        - else
+          li.nav-item: a.nav-link[href='/users/sign_in'] Sign In
```
Comment (andrjuha01) [inline app/views/shared/_header.html.slim:24]: link_to
Same in a row below

File: app/views/shared/_header.html.slim [hunk 6]
```diff
@@ -5,8 +5,22 @@ header.d-flex.w-100.flex-column
...
         li.nav-item: a.nav-link[href='#'] Contact
     = link_to "HostMePlease", root_path, class: "navbar-brand mx-auto nav-link"
-    div.collapse.navbar-collapse
+    div.collapse.navbar-collapse.userinfo
       ul.navbar.navbar-nav.ms-auto
-        li.nav-item: a.nav-link[href='/users/sign_in'] Sign In
-        li.nav-item: a.nav-link[href='/users/sign_up'] Sign Up
+        - if user_signed_in?
+          li.nav-item.dropdown
+            a.nav-link#dropdownMenuLink.dropdown-toggle aria-expanded="false" data-bs-toggle="dropdown" role="button"
```
Comment (andrjuha01) [inline app/views/shared/_header.html.slim:12]: First 'id', than classes

File: app/frontend/stylesheets/header.scss (hunk 1 above)
Comment (Hunter2065) [reply app/frontend/stylesheets/header.scss:29]: okay), I must remove it?

File: app/frontend/stylesheets/header.scss (hunk 2 above)
Comment (Hunter2065) [reply app/frontend/stylesheets/header.scss:33]: okay

File: app/helpers/users_helper.rb (hunk 3 above)
Comment (Hunter2065) [reply app/helpers/users_helper.rb:2]: sorry, forget change it

File: app/helpers/users_helper.rb (hunk 4 above)
Comment (Hunter2065) [reply app/helpers/users_helper.rb:4]: fixed

File: app/helpers/users_helper.rb (hunk 3 above)
Comment (Hunter2065) [reply app/helpers/users_helper.rb:2]: fixed

File: app/views/shared/_header.html.slim (hunk 5 above)
Comment (Hunter2065) [reply app/views/shared/_header.html.slim:24]: fixed

File: app/views/shared/_header.html.slim (hunk 6 above)
Comment (Hunter2065) [reply app/views/shared/_header.html.slim:12]: fixed

**Review by andrjuha01 — APPROVED** [review]

**Pull request merged**